python/tests/res/job_queue/test_forward_model_formatted_print.py

```diff
@@ -341,15 +342,44 @@ def test_no_jobs(self):
             run_id = "test_no_jobs_id"
             umask = 4
             global_args = SubstitutionList()
+            varlist = EnvironmentVarlist()
             forward_model.formatted_fprintf(
                 run_id,
                 os.getcwd(),
                 "data_root",
                 global_args,
-                umask)
+                umask,
+                varlist)
 
             self.verify_json_dump([], global_args, umask, run_id)
 
+    def test_env_varlist(self):
+        varlist_string = "global_environment"
+        first = "FIRST"
+        second = "SECOND"
+        first_value = "TheFirstValue"
+        second_value = "TheSecondValue"
+        varlist = EnvironmentVarlist()
```

Comment:
Please add a third variable:
```
third = "THIRD"
third_value = "$first:$second"
```
and then the test:

```python
self.assertEqual("%s:%s" % (first_value, second_value), env_config[third] )
```
this will test that the `util_setenv_interp()` function has been used correctly; which is not the case right now.